modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,344 @@
+
+* Changed operation names, metadata, fields, and values
++
+*Reviewers, what values were changed? I do not see them in the PR. Also, were some operations added besides the Object Listener input source?*
++
+* Changed input sources
+* Changed connection types
+
+=== Changed Operation Names
+
+The following operation names changed in MongoDB 6.0.0:
+
+[%header]
+|===
+|5.3.x|6.0.0
+|Map reduce objects |Map reduce
+|Exists collection |Collection exists
+|List indices |List indexes
+|===
+
+== Changed Operations Parameters
+
+[%header,cols="19%,15%,15%,19%,15%,15%"]
+|===
+3+|Existing in MongoDB 5.x 3+| Modified in MongoDB 6.x
+|Operation Name | Input Params  |Output Params  | Operation Name                             |Input Params                                      |Output Params
+| Drop database
+| N/A
+| N/A
+| No changes
+a|* Database name (String)
+| N/A
+| Dump
+a|* Id (String)
+* OutputDirectory (String)
+* OutputName (String)
+* Zip (Boolean)
+* Oplog (Boolean)
+* Threads(Int)
+| N/A
+| No changes
+a|* OutputDirectory (String)
+* OutputNamePrefix (String)
+* Oplog (No Change)
+* OperationTimeout (Integer)
+* OperationTimeoutUnits (Java TimeUnit)
+| N/A
+| Add user
+a| * Username (String)
+* Password (String)
+| N/A
+| No changes
+a| * Username (String)
+* Password (String)
+| N/A
+| Create Collection
+a| * Collection name (String)
+* Capped (Boolean)
+* Max objects (Number)
+* Collection size (Number)
+| N/A
+| No changes
+a| * Collection name (String)
+* Capped options (Object)
+* Max objects (Number: as part of CappedOptions)
+* Collection size (Number: as part of CappedOptions)
+* Collection size data unit (Number: as part of CappedOptions)
+| N/A
+| Drop collection
+a|* Collection name (String)
+| N/A
+| No changes
+a|* Collection name (String)
+|N/A
+| Exists collection
+a|* Collection name (String)
+| N/A
+| Collection exists
+a|* Collection name (String)
+| N/A
+| List collections
+| N/A
+| N/A
+| No changes
+| N/A
+| N/A
+| Map reduce objects
+a|* Collection name (String)
+* Mapping function (String)
+* Reduce function (String)
+* Output collection (String)
+| N/A
+| Map reduce
+a|* Collection name (String)
+* Mapping function (String)
+* Reduce function (String)
+* Output collection (String)
+| N/A
+| Create index
+a|* Collection name (String)
+* Field name (String)
+* Order (Enum: ASC or DESC)
+| N/A
+| No changes
+a|* Collection name (String)
+* Field name (String)
+* Sort order (Enum: ASC or DESC)
+| N/A
+| Drop index
+a|* Collection name (String)
+* Index name (String)
+| N/A
+| No changes
+a|* Collection name (String)
+* Index name (String)
+| N/A
+| List indices
+a|* Collection name (String)
+| N/A
+| List indexes
+a|* Collection name (String)
+| N/A
+| Insert document
+a|* Collection name (String)
+* Document (InputStream)
+| N/A
+| No changes
+a|* Collection name (String)
+* Document (InputStream)
+| N/A
+| Insert documents
+a|* Collection name (String)
+* Documents (InputStream)
```

Comment:
For the user, it's a JSON Array with the documents. Please keep it as java agnostic as possible

Reply:
please remove InputStream usage on the whole document